packages/happy-dom/src/dom-token-list/DOMTokenList.ts

```diff
@@ -212,6 +204,18 @@ export default class DOMTokenList implements IDOMTokenList {
 		(<number>this.length) = list.length;
 	}
 
+	/**
+	 * Returns token list from attribute value.
+	 *
+	 * @see https://infra.spec.whatwg.org/#split-on-ascii-whitespace
+	 */
+	private _getTokenList(): string[] {
+		let attr = this._ownerElement.getAttribute(this._attributeName) ?? '';
+		attr = attr.replace(/^[\t\n\f\r ]+|[\t\n\f\r ]+$/g, '');
+		attr = attr.replace(/[\t\n\f\r ]+/g, ' ');
```

Comment:
Hard to demonstrate as I’m typing on mobile, but String.prototype.split() accepts RegExp so this replace() and the next split() could both be reduced to one step.

Reply:
Thank you for the review. I have merged the two lines.

I learned a lot this time that String.prototype.split() accepts RegExp. Thank you again.